app/src/main/java/com/courseproject/tindar/entities/Filters.java

```diff
@@ -0,0 +1,23 @@
+package com.courseproject.tindar.entities;
+
+public class Filters {
+    private final int preferredAgeMinimum;
```

Comment:
making these final might be a bad idea depending on whether we're updating filters by updating the object, or making  anew onw.

Reply:
I probably made it because Android studio complained it can be made to final. Either way, the current usage of Filter class to check the age group is valid and the object instantiated wouldn't be updated / needed after the check. A new object will be created everytime the user is updating age group.